docs/management/index-patterns.asciidoc

```diff
@@ -89,6 +89,11 @@ To query across all {es} clusters that have been configured for {ccs},
 use a standalone wildcard for your cluster name in your index
 pattern: `*:logstash-*`.
 
+Exclusions can be used too.
```

Comment:
```suggestion
Exclusions can be used to not include indices which might contain mapping errors.
```

I feel saying "narrow the number of indices" might be an anti-pattern. Having a lot of indices to search on should not be a problem.

Reply:
How about:

You can use exclusions to exclude indices that might contain mapping errors.